R/epidist.R

```diff
@@ -948,14 +960,17 @@ is_truncated <- function(x) {
 #' @examples
 #' edist <- epidist_db(
 #'   disease = "COVID-19",
-#'   epi_dist = "incubation_period",
+#'   epi_dist = "incubation period",
 #'   single_epidist = TRUE
 #' )
 #' mean(edist)
 mean.epidist <- function(x, ...) {
-
   # extract mean if given
-  mean <- x$summary_stats$centre_spread$mean
+  if (utils::hasName(x$summary_stats, "mean")) {
+    mean <- x$summary_stats$mean
+  } else {
+    return(NA)
```

Comment:
Horrendous nitpicking here, but consider `NA_real_`

Reply:
Resolved in ed39ed92d8cf97fe6b7b421d92cd3261398bcdf9.